src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -37,8 +37,7 @@ package object servlets extends DebugEnhancedLogging {
   private val post = "-compress(ed)?"
   private val contentTypeZipPattern = s"application/(($pre($post)?)|(x$post))"
 
-  val contentTypeJson: (String, String) = "content-type" -> "application/json;charset=UTF-8"
-  val contentTypePlainText: (String, String) = "content-type" -> "text/plain;charset=UTF-8"
+  private val contentTypePlainText: (String, String) = "content-type" -> "text/plain;charset=UTF-8"
```

Comment:
Is this one still used?

Reply:
Only a few lines below:
https://github.com/DANS-KNAW/easy-deposit-api/blob/7f7fcd1699c908c8781eaa7baba971713c446bd8/src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala#L40-L50